### DISCUSSION TITLE
分叉后直接Action构建失败，请问如何用Action自己构建

### DISCUSSION THREAD

**Discussion by vqzhanshi** [zhongfly/mpv-build#63] [Q&A] [answered]
CMake Error at build64-v3/cmake/Modules/ExternalProject.cmake:4307 (cmake_policy):
  cmake_policy GET given policy "CMP0150" which is not known to this version
  of CMake.
Call Stack (most recent call first):
  toolchain/mingw-w64.cmake:1 (ExternalProject_Add)
  toolchain/CMakeLists.txt:17 (include)

**Comment by zhongfly** [comment, marked as answer]
https://github.com/shinchiro/mpv-winbuild-cmake/issues/365

那个项目的问题吧，大概是cmake版本的问题？

**Comment by vqzhanshi** [comment]
我在fork您的项目后未做其他更改，直接action后提示该错误然后不能执行下去。
请问action的话还需要其他修改吗，我研究了下mpv.yml文档，没有找到需要修改的地方
您的项目仍然正常编译发布，说明不是上游项目的问题
我是个小白，是不是我的操作有问题，在搜索一些答案后还是不能解决
希望能得到您的指点

**Reply by zhongfly** [reply]
前面说过原因了，我能运行是因为之前已经完成了相关工具链的构建，现在使用缓存、没有重复构建会出问题的那部分
既然你看得懂那你也应该知道与本项目无关

**Reply by zhongfly** [reply]
https://github.com/shinchiro/mpv-winbuild-cmake/issues/274
类似的情况，要不等上游更新，要不你自己想办法用其他版本cmake，自行谷歌解决。

**Reply by vqzhanshi** [reply]
哦，我忽略了缓存，感谢！

**Reply by zhongfly** [reply]
https://github.com/zhongfly/mpv-winbuild/tree/cmake
先用这个分支吧，追根溯源cmake项目那边的问题
